src/base/system_parameter.c

```diff
@@ -3623,7 +3623,7 @@ static SYSPRM_PARAM prm_Def[] = {
    (DUP_PRM_FUNC) NULL},
   {PRM_ID_LIST_MAX_QUERY_CACHE_ENTRIES,
    PRM_NAME_LIST_MAX_QUERY_CACHE_ENTRIES,
-   (PRM_FOR_SERVER),
+   (PRM_FOR_SERVER | PRM_FORCE_SERVER),
```

Comment:
PRM_ID_LIST_MAX_QUERY_CACHE_ENTRIES and PRM_ID_LIST_MAX_QUERY_CACHE_PAGES should apply PRM_HIDDEN same like PRM_ID_LIST_QUERY_CACHE_MODE?

Reply:
PRM_ID_LIST_MAX_QUERY_CACHE_ENTRIES and PRM_ID_LIST_MAX_QUERY_CACHE_PAGES should be allowed to disable the "query cache mode". It is a policy.

Reply:
ok, I understand.